base/src/worksheet.rs

```diff
@@ -522,6 +522,23 @@ impl Worksheet {
         Ok(is_empty)
     }
 
+    /// Returns true if cell part of Merged Cells.
+    /// First cell of Merge block is not considered as part of Merge cell
+    /// Ex : if Merge cells were A1-C3, A1 is not considered as part of Merge cell
+    pub fn is_part_of_merge_cell(&self, row: i32, column: i32) -> Result<bool, String> {
+        if !is_valid_column_number(column) || !is_valid_row(row) {
+            return Err("Incorrect row or column".to_string());
+        }
+
+        //traverse through Vector of Merged Cells and return (linear search)
```

Comment:
Please always leave a space between the `//` and the rest of the comment. I will try to have a linter that imposes that rule.